### PR TITLE
Rs demographics modeler

### PR DESCRIPTION
**Pull request by riaansnyders**
Please review. Demograhics Modeller data binding completed

### CODE REVIEW

File: scoda/templates/scoda.html [hunk 1]
```diff
@@ -10,6 +10,10 @@
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
+     <!--Leaflet for maps-->
+     <link rel="stylesheet" href="https://unpkg.com/leaflet@1.0.2/dist/leaflet.css"/>
```
Comment (callum09) [inline scoda/templates/scoda.html:14]: @riaansnyders could you please keep leaflet in the gulp js bundle. It helps us with load times.

File: scoda/templates/scoda.html [hunk 2]
```diff
@@ -20,6 +24,75 @@
     <link rel="stylesheet" href="/static/dist/fonts/fontawesome-free/css/all.min.css" />
     <link href="https://fonts.googleapis.com/css2?family=Megrim&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="/static/dist/css/ds-style.css" />
+    <style>
```
Comment (callum09) [inline scoda/templates/scoda.html:27]: @riaansnyders please move these styles into their own sass partial, either new or existing. As a general rule we don't use inline styles, unless unavoidable.

Comment (riaansnyders) [inline scoda/templates/scoda.html:27]: Moved into demograhics sass. Will be in next push.

File: scoda/templates/scoda.html (hunk 1 above)
Comment (riaansnyders) [reply scoda/templates/scoda.html:14]: Completed. Will be in next push.

**Pull request merged**